app/Http/Controllers/CommunityController.php

```diff
@@ -65,8 +65,9 @@ public function getLive(Request $request)
 
         $featuredStreamId = Cache::get('featuredStream');
         if ($featuredStreamId !== null) {
+            $featuredStreamId = floatval($featuredStreamId);
             foreach ($streams as $stream) {
-                if ($stream->_id !== $featuredStreamId) {
+                if (floatval($stream->_id) !== $featuredStreamId) {
```

Comment:
Yeah, I think depending on cache driver, it casts to string. I also casted _id to float (but now I checked it, it is already float), just to be safe.